user-manual/transfer/dataverse.rst

```diff
@@ -0,0 +1,118 @@
+* :ref:`Configuration <dataverse-configuration>`
+
+.. _selecting-datasets:
+
+Selecting Datasets for preservation
+-----------------------------------
+
+When a Dataverse Transfer Source Location is selected in the Transfer Tab of 
+the Dashboard, users can browse a list of available datasets. Selecting a 
+directory icon will expand the view to display the list of files included in 
+the dataset. Individual files can’t be selected for Transfer.
+
+When a Dataverse dataset is selected, the Transfer Type ‘Dataverse’ must also 
+be selected.
+
+.. _processing-datasets:
+
+Processing Dataverse datasets
+-----------------------------
+
+Archivematica creates a Dataverse METS file to describe the contents and 
+structure of the dataset as retrieved from Dataverse. Archivematica also 
+creates an agents.json file, that includes details of the Dataverse instance 
+configured in the Storage Service. This information is used to populate the 
+Dataverse PREMIS agent details in the AIP METS. 
+
+Fixity checks are conducted using any checksums provided by Dataverse. Other 
+microservices are carried out as normal (and as configured in the processing 
+configuration). The final AIP will contain descriptive metadata provided by 
+Dataverse, attributes to indicate any derivatives generated by Dataverse, 
+and attributes to indicate the outcome of fixity checks conducted using 
+checksums provided by Dataverse.
+
+.. _dataverse-mets:
+
+Dataverse METS file
+-------------------
+
+Archivematica generates a Dataverse METS file that describes the contents of 
+the dataset as retrieved from Dataverse. The Dataverse METS includes:
+
+* descriptive metadata about the dataset, mapped to the `DDI standard`_
+
+* a <mets:fileSec> section that lists all files provided, grouped by type 
+  (original, metadata or derivative)
+
+* a <mets:structMap> section that describes the structure of the files as 
+  provided by Dataverse (particularly helpful for understanding which files were
+  provided in `Tabular Data file bundle`_)
+
+The Dataverse METS is found in the final AIP in this location: 
+<AIP Name>/data/objects/metadata/transfers/<transfer name>/METS.xml 
+(This is also where you will find the dataset.json metadata file provided by 
+Dataverse, and the agents.json metadata file created by Archivematica).
+
+.. _dataverse-aip-mets:
+
+AIP METS file
+-------------
+
+The Archival Information Package (AIP) METS file follows the basic structure 
+for a standard Archivematica AIP METS file. Derivatives generated by Dataverse
+are indicated using the METS fileGrp attribute (where USE =“derivative”).
+
+The descriptive metadata (dmdSecs) in the Dataverse METS file are copied over 
+to the AIP METS file.
+
+In the PREMIS Object entity, relationships between original and derivative 
+tabular format files from Dataverse are described using PREMIS relationship 
+semantic units. A PREMIS derivation event indicates the derivative file was 
```

Comment:
Possibly just need `... PREMIS semantic units.`.

Reply:
yes, that does read better. (less is more)